src/onboarding/Create/Create.js

```diff
@@ -509,6 +511,31 @@ const Create = React.memo(function Create({
     [web3]
   )
 
+  const applyEstimatePriorityFee = useCallback(
```

Comment:
@mathewmeconry If the function is exactly the same thing (seems to be), please move it to some helper, and import in both places

Reply:
done